src/lib/consensus/proof_of_stake.ml

```diff
@@ -2420,7 +2420,8 @@ module Make_str (A : Wire_types.Concrete) = struct
       module Unsafe = struct
         (* TODO: very unsafe, do not use unless you know what you are doing *)
         let dummy_advance (t : Value.t) ?(increase_epoch_count = false)
-            ~new_global_slot_since_genesis : Value.t =
+            ?(increase_blockchain_length = false) ~new_global_slot_since_genesis
```

Comment:
We should probably refrain from doing stuff like this if we can to improve compatibility with upstream,
maybe copying/inlining the entire thing is better, especially since this function seems small (unless it operates on unexported definitions?)

Reply:
Yes it operates on unexported definitions. I guess the reason is for it to have a modular consensus.